[2025-05-08T12:10:02.806+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-08T12:10:02.813+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.weather_dag_1 scheduled__2025-05-08T12:05:00+00:00 [queued]>
[2025-05-08T12:10:02.817+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.weather_dag_1 scheduled__2025-05-08T12:05:00+00:00 [queued]>
[2025-05-08T12:10:02.817+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-08T12:10:02.829+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): weather_dag_1> on 2025-05-08 12:05:00+00:00
[2025-05-08T12:10:02.831+0000] {standard_task_runner.py:72} INFO - Started process 31187 to run task
[2025-05-08T12:10:02.834+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'weather_dag_1', 'scheduled__2025-05-08T12:05:00+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpnm0_oh_a']
[2025-05-08T12:10:02.835+0000] {standard_task_runner.py:105} INFO - Job 167: Subtask weather_dag_1
[2025-05-08T12:10:02.862+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_dag.weather_dag_1 scheduled__2025-05-08T12:05:00+00:00 [running]> on host codespaces-16c72d
[2025-05-08T12:10:02.907+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='weather_dag_1' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T12:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T12:05:00+00:00'
[2025-05-08T12:10:02.907+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-08T12:10:02.920+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-08T12:10:02.920+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'bash /workspaces/airflow/dags/wrapper.sh ']
[2025-05-08T12:10:02.927+0000] {subprocess.py:99} INFO - Output:
[2025-05-08T12:10:02.930+0000] {subprocess.py:106} INFO - /workspaces/airflow/dags/wrapper.sh: line 2: weather/bin/activate: No such file or directory
[2025-05-08T12:10:03.909+0000] {subprocess.py:106} INFO - 002002 (42710): SQL compilation error:
[2025-05-08T12:10:03.910+0000] {subprocess.py:106} INFO - Object 'WEATHER_API' already exists.
[2025-05-08T12:10:03.910+0000] {subprocess.py:106} INFO - ('9.11.3',)
[2025-05-08T12:10:03.984+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-08T12:10:03.998+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-08T12:10:03.998+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=weather_dag_1, run_id=scheduled__2025-05-08T12:05:00+00:00, execution_date=20250508T120500, start_date=20250508T121002, end_date=20250508T121003
[2025-05-08T12:10:04.049+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-08T12:10:04.057+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-08T12:10:04.058+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
